viewer/packages/rendering/src/glsl/post-processing/blit.frag

```diff
@@ -0,0 +1,73 @@
+precision highp float;
+
+uniform sampler2D tDiffuse;
+
+#if defined(DEPTH_WRITE) 
+uniform sampler2D tDepth;
+#endif
+
+#if defined(SSAO_BLUR) 
+uniform sampler2D tSsao;
+#endif
+
+#if defined(ALPHA)
+uniform float alpha;
+#endif
+
+#if defined(OUTLINE)
+uniform sampler2D tOutlineColors;
+#endif
+
+in vec2 vUv;
+
+out vec4 fragColor;
+
+#if defined(SSAO_BLUR) 
+#pragma glslify: import('./gaussian-blur.glsl')

```

Comment:
Can't have combine with with respective `uniform` declaration?

Reply:
This would brake the "fix" you did since uniforms apparently needs to come before any function definition on iOS. This may of course have been fixed later, but even then it is cleaner to have all the uniforms defined before functions.

Reply:
Also the checks here are only compiler checks, so it should not effect runtime performance.